src/components/PersonPage/PersonPage.tsx

```diff
@@ -0,0 +1,52 @@
+import { useParams } from 'react-router-dom';
+import { memo } from 'react';
+import cn from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  person: Person,
+  people: Person[],
+};
+
+export const PersonPage: React.FC<Props> = memo(({ person, people }) => {
+  const { slug = '' } = useParams();
+
+  const isSelected = (selectedperson: Person) => selectedperson.slug === slug;
+
+  const findParent = (name: string) => {
+    const parent = people.find(pers => pers.name === name);
+
+    return parent
+      ? <PersonLink person={parent} />
+      : name;
+  };
```

Comment:
U could prepare people after loading and set into state people with mother and father.